integrations/sonarqube/README.md

```diff
@@ -0,0 +1,17 @@
+# SonarQube integration
+
+A [jq](https://stedolan.github.io/jq/) filter is provided in [mutation-report-to-sonar.jq](mutation-report-to-sonar.jq) to convert a JSON [mutation testing report](../../packages/report-schema/) to the [SonarQube generic issue import format](https://docs.sonarqube.org/latest/analysis/generic-issue/).
+
+Usage:
+
+```
+jq -f mutation-report-to-sonar.jq mutation.json > mutation-sonar.json
+```
+
+After `mutation-sonar.json` is generated, you can import it into SonarQube using the analysis parameter `sonar.externalIssuesReportPaths`. See [Generic Issue Import Format](https://docs.sonarqube.org/latest/analysis/generic-issue/) for more details.
+
+Please note that:
```

Comment:
You should also add a note with:

"
There are a couple of limitations with importing external issues:

you can't manage them within SonarQube; for instance, there is no ability to mark them False Positive.
you can't manage the activation of the rules that raise these issues within SonarQube. External rules aren't visible on the Rules page or reflected in Quality Profiles.
External issues and the rules that raise them must be managed in the configuration of your linter.
"
See: https://docs.sonarqube.org/latest/analysis/generic-issue/